iOS Dev Calendar/Calendar/Event/EventsView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  EventsView.swift
+//  iOS Dev Calendar
+//
+//  Created by Wesley Keetch on 4/22/25.
+//
+
+
+import SwiftUI
+
+struct EventsView: View {
+  let date: Date
+  let events: [Date: [Event]]
```

Comment:
Why does this structure have a date and then a dictionary of `date: [Event]`?
 Shouldn't it just be a date and then array of `Event` associated with that date?